tsconfig.json

```diff
@@ -19,7 +19,10 @@
     "noImplicitAny": false,
     "noFallthroughCasesInSwitch": true,
     "downlevelIteration": true,
-    "incremental": true
+    "incremental": true,
+    "paths": {
+      "@pancakeswap/uikit": ["../packages/uikit/src/index.ts"]
+    }
```

Comment:
If you are looking for something deeper. This is where magic happens. 
mapping src instead of as node module `dist` and avoid parallel `dev` scripts